src/main/java/seedu/address/logic/commands/LinkCommand.java

```diff
@@ -84,12 +84,12 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(String.format(
                     MESSAGE_LINKED_PERSON,
                     personToLink.getName(),
-                    model.findInternshipById(personToLink.getInternshipId()).getCompanyName()));
+                    model.findInternshipById(personToLink.getInternshipId()).getDisplayName()));
         } else if (internshipToLink.getContactPersonId() != null) {
             throw new CommandException(String.format(
                     MESSAGE_LINKED_INTERNSHIP,
```

Comment:
Likewise for line 37:
`MESSAGE_LINKED_INTERNSHIP = "%2$s is already linked to %1$s"`